docs/getting_started/integration_tests.md

```diff
@@ -0,0 +1,150 @@
+		Writer:  gosoFixtures.ConfigurableKvStoreFixtureWriterFactory("currency"),
+		Fixtures: []interface{}{
+			&gosoFixtures.KvStoreFixture{
+				Key:   "GBP",
+				Value: 1.23,
+			},
+			&gosoFixtures.KvStoreFixture{
+				Key:   "2021-01-03-GBP",
+				Value: 1.24,
+			},
+		},
+	},
+}
+```
+
+This _FixtureSet_ holds exchange rate data for GBP, which will be needed by the test.
+
+`api_test.go` declares _ApiTestSuite_ which will implement _TestingSuite_ :
+
+[embedmd]:# (../../examples/getting_started/integration/api_test.go /type ApiTestSuite/ /\n}/)
+```go
+type ApiTestSuite struct {
+	suite.Suite
+
+	client *resty.Client
+	clock  clock.Clock
+}
+```
+
+It also declares a normal unit test which makes use of the above struct, and calling _suite.Run_:
+
+[embedmd]:# (../../examples/getting_started/integration/api_test.go /func TestApiTestSuite/ /\n}/)
+```go
+func TestApiTestSuite(t *testing.T) {
+	suite.Run(t, &ApiTestSuite{
+		clock: clock.NewFakeClockAt(time.Now().UTC()),
+	})
+}
+```
+
+Another important thing about _ApiTestSuite_ is how it configures the kernel which will run the tests:
+
+[embedmd]:# (../../examples/getting_started/integration/api_test.go /func \(s \*ApiTestSuite\) SetupSuite/ /\n}/)
+```go
+func (s *ApiTestSuite) SetupSuite() []suite.Option {
+	clock.WithUseUTC(true)
+	clock.Provider = s.clock
+
+	return []suite.Option{
+		suite.WithLogLevel("info"),
+		suite.WithConfigFile("../api/config.dist.yml"),
+		suite.WithFixtures(fixtureSets),
+		suite.WithClockProvider(s.clock),
+	}
+}
+```
+
+In `SetupSuite` it makes use of the same `config.dist.yml` file that _money-exchange_ uses, and also of the fixture data declared in `fixtures.go`.
+
+The api server itself is configured by `SetupApiDefinitions`:
+[embedmd]:# (../../examples/getting_started/integration/api_test.go /func \(s \*ApiTestSuite\) SetupApiDefinitions/ /\n}/)
+```go
+func (s *ApiTestSuite) SetupApiDefinitions() apiserver.Definer {
+	return definer.ApiDefiner
+}
+```
+
+Lastly, for each method of _ApiTestSuite_ starting with `Test...`, Gosoline will run a separate test case. In this example there is only one such method:
+
+[embedmd]:# (../../examples/getting_started/integration/api_test.go /func \(s \*ApiTestSuite\) Test_API/ /\n}/)
+```go
+func (s *ApiTestSuite) Test_API(_ suite.AppUnderTest, client *resty.Client) error {
+	s.client = client
+
+	s.testToEuro()
+	s.testToEuroAtDate()
+
+	return nil
+}
+```
+
+Both `testToEuro` and `testToEuroAtDate` will issue a REST call to the API server, and check its response for correctness. Below is the code for `testToEuro`:
+
+[embedmd]:# (../../examples/getting_started/integration/api_test.go /func \(s \*ApiTestSuite\) testToEuro/ /\n}/)
+```go
+func (s *ApiTestSuite) testToEuro() {
+	var result float64
+
+	response, err := s.client.R().
+		SetBody(`{}`).
+		SetResult(&result).
+		Execute(http.MethodGet, "/euro/10/GBP")
+
+	s.NoError(err)
+	s.Equal(http.StatusOK, response.StatusCode())
+	s.Equal(8.130081300813009, result)
```

Comment:
I think I would prefer this number to be... better. Like provide nice, round numbers in the fixtures so we also get a nice number here